### DISCUSSION TITLE
Restore deleted codespaces

### DISCUSSION THREAD

**Discussion by vedantmgoyal9** [community/community#12112] [Codespaces] [answered]
I have deleted a repository. It was forked by users so it can't be recovered. The problem is that I had configured more than 1 repository in the codespace linked to that repository and on deleting the repository, the codespace was also deleted automatically. There were so many uncommitted changes that weren't pushed to other repositories. How to recover that codespace, anyone from @github team, please help! I had created a support ticket, but nobody is responding. Please help🙏

**Comment by joshspicer** [comment, marked as answer]
I believe we were able to get to a resolution on your support ticket, yes? If not, please let us know!
